packages/components/src/field-group/FieldGroup.js

```diff
@@ -22,10 +22,10 @@ const FieldGroup = ( { htmlFor, label, linkTo, linkText, description, children }
 		<div className="yoast-field-group">
 			<div className="yoast-field-group__title">
 				<label htmlFor={ htmlFor }>{ label }</label>
-				<HelpIcon
+				{ ( !! linkTo ) && <HelpIcon
```

Comment:
I think
```suggestion
				{ linkTo !== "" && <HelpIcon
```
would be a little more obvious.